chapters/pi/car/car-face.md

```diff
@@ -477,7 +477,7 @@ provide as Human Face Detected from the capturing video.
     ## Create the haar cascade
     faceCascade = cv2.CascadeClassifier(cascPath)
 
-Camera settings need to be updated in the code as per below suggestions.
+Camera settings need to be updated in the code as per suggestions next.
```

Comment:
Camera settings need to be updated, our suggested changes are shown in the code next.